src/ifcopenshell-python/test/fixtures/validate/pass-valid-selected-enumeration.ifc

```diff
@@ -5,6 +5,8 @@ FILE_NAME('','2023-04-13T10:27:43',(),(),'IfcOpenShell v0.7.0-198fa67cc','IfcOpe
 FILE_SCHEMA(('IFC4X3_RC4'));
 ENDSEC;
 DATA;
-#1=IFCFACILITYPART('0DYKeUG9993PtW$2icoR4v',$,$,$,$,$,$,$,$,IFCBRIDGEPARTTYPEENUM('DECK'),.LATERAL.);
```

Comment:
@I-Sokolov I think this was correct (and created specifically for this purpose). This construct doesn't exist anymore in IFC4X3 now, but in RC4 we had this idea of concrete facility part with the select of enumeration. See https://github.com/buildingSMART/IFC4.3.x-development/issues/90